x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/analysis/analyzer/Verifier.java

```diff
@@ -266,6 +268,14 @@ public Verifier(Metrics metrics) {
         return failures;
     }
 
+    private void checkNestedAggregation(LogicalPlan p, Set<Failure> localFailures, AttributeMap<Expression> attributeRefs) {
+        if(p instanceof Aggregate){
+            ((Aggregate) p).child().forEachDown(Aggregate.class, a -> {
+                localFailures.add(fail(a, "Nested aggregations in sub-selects are not (yet) supported."));
```

Comment:
I'd drop the `(yet)`.

Reply:
👍 I saw this phrase in a couple of other places but agree that it's not really necessary.